src/effects/browse.js

```diff
@@ -4,9 +4,11 @@ import { CURRENT_TERM } from 'util/constants';
 const currentTermDoc = db.collection('terms').doc(CURRENT_TERM);
 
 /* eslint-disable import/prefer-default-export */
-export function fetchSchools() {
```

Comment:
We still need the `fetchSchools` effect.

Reply:
Oops I accidentally deleted that